manifest/content.xml

```diff
@@ -73,4 +73,27 @@
     <button />
   </form>
 
+  <!--
+    Tabs
+      Required childr elements: options, option, tab-content, tab
+      Each option element requires a single content:text child element
+      The number of tab elements should be the same of the number of option element
+  -->
+  <tabs>
+    <tab>
+      <label>
+        <text i18n-id="{{uuid}}">Option A</text>
```

Comment:
all of the XML elements in this specific file are in the same xml namespace as `content:*` in other documents. XML namespace prefixes are only used to lookup the namespace URI in the xml document they are in. Prefixes can be any string you want within that xml file as long as you assign them to the correct namespace URI :)